\section{クラス ModuleManager::Error}
\label{classsource__py_1_1_module_manager_1_1_module_manager_1_1_error}\index{source\_\-py::ModuleManager::ModuleManager::Error@{source\_\-py::ModuleManager::ModuleManager::Error}}
ファイル・オープン失敗例外処理用内部クラス  


{\bf ModuleManager::InvalidArguments}, {\bf ModuleManager::InvalidOperation}, と {\bf ModuleManager::NotAllowedOperation}に継承されています。



\subsection{説明}
ファイル・オープン失敗例外処理用内部クラス 

 ModuleManager.py の 125 行で定義されています。